src/test/java/io/neonbee/internal/handler/factories/CorrelationIdHandlerFactoryTest.java

```diff
@@ -0,0 +1,37 @@
+package io.neonbee.internal.handler.factories;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.MockedStatic;
+
+import io.neonbee.NeonBee;
+import io.neonbee.config.ServerConfig;
+import io.neonbee.internal.handler.CorrelationIdHandler;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+
+@ExtendWith(VertxExtension.class)
+class CorrelationIdHandlerFactoryTest {
+
+    @Test
+    void testCreateHandler(VertxTestContext testContext) {
+
+        try (MockedStatic<NeonBee> staticNeonBeeMock = mockStatic(NeonBee.class)) {
+            NeonBee neonBeeMock = mock(NeonBee.class);
+            when(neonBeeMock.getServerConfig()).thenReturn(new ServerConfig());
```

Comment:
I think you could also also work here with the NeonBeeMockHelper in order to avoid the usage of mockStatic. Not required to change this, but generally I think it's more convenient to use the helper.